content/applications/essentials/in_app_purchase.rst

```diff
@@ -1,57 +1,186 @@
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
+   denominations of:
+
+   - :guilabel:`Starter Pack`: 10 credits
+   - :guilabel:`Standard Pack`: 100 credits
+   - :guilabel:`Advanced Pack`: 500 credits
+   - :guilabel:`Expert Pack`: 1,000 credits
+
+   .. image:: in_app_purchase/packs.png
+      :align: center
+      :alt: Four different packs of credits for the SMS IAP service.
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+
+   For more information, refer to the :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
+
+.. _iap/buying_credits:
+
+Buy credits
+-----------
+
+If there are not enough credits to perform a task, the database automatically prompts the purchase
+of more credits.
+
+Users can check the current balance of credits for each service, and manually purchase more credits,
+by navigating to the :menuselection:`Settings app --> Contact --> Odoo IAP --> View My Services`.
+
+Doing so reveals an :guilabel:`IAP Account` page, listing the various |IAP| services in the
+database. From here, click an |IAP| service to open its :guilabel:`Account Information` page, where
+additional credits can be purchased.
+
+Manually buy credits
+~~~~~~~~~~~~~~~~~~~~
+
+To manually buy credits in Odoo, follow these steps:
+
+First, go to the :menuselection:`Settings application` and type `IAP` in the search bar. Under the
+Contacts section, where it says :guilabel:`Odoo IAP`, click :guilabel:`View My Services`.
+
+   .. image:: in_app_purchase/view-services.png
+      :align: center
+      :alt: The Settings app showing the Odoo IAP heading and View My Services button.
+
+Doing so reveals an :guilabel:`IAP Account` page, listing the various |IAP| services in the
+database. From here, click an |IAP| service to open details about it; additional credits can be
+purchased from here.
+
+On the following page, click the :guilabel:`Buy Credit` button. Doing so loads a :guilabel:`Buy
+Credits for (IAP Account)` page in a new tab. From here, click :guilabel:`Buy` on the desired pack
+of credits. Then, follow the prompts to enter payment details, and confirm the order.
+
+   .. image:: in_app_purchase/buy-pack.png
+      :align: center
+      :alt: The SMS service page on IAP.Odoo.com with four packs of credits available for purchase.
+
+Once the transaction is complete, the credits are available for use in the database.
+
+.. _in_app_purchase/low-credits:
+
+Low-credit notification
+~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to be notified when credits are low, in order to avoid running out of credits, while
+using an |IAP| service. To do that, follow this process:
+
+Go to the :menuselection:`Settings application` and type `IAP` in the search bar. Under the Contacts
+section, where it says :guilabel:`Odoo IAP`, click :guilabel:`View My Services`.
```

Comment:
i think Contacts can have a GUI because it is ON the UI. If not, I would italicize it.